ulmo/waterml/common.py

```diff
@@ -139,7 +139,7 @@ def _element_dict(element, exclude_children=None, prepend_attributes=True):
     element_dict.update(dict([
         (_element_dict_attribute_name(key, element_name,
             prepend_element_name=prepend_attributes), value)
-        for key, value in element.attrib.iteritems()
+        for key, value in element.attrib.items()
```

Comment:
That explains some other bugs I had to fix where it changed iteritems to items on pandas dataframes. Thanks for the catch.